test/simulation.spec.ts

```diff
@@ -94,4 +97,24 @@ describe('Craft simulator tests', () => {
     expect(simulation.quality).toBe(626); // starting HQ with 1 each of oyster+lemonette for 5481
     expect(simulation.success).toBeFalsy();
   });
+
+  it('Should fail a craft when user is below minimum stat requirements', () => {
+    const simulation = new Simulation(
+      generateStarRecipe(480, 4943, 32328, 2480, 2195),
+      [
+        new MuscleMemory(),
+        new WasteNot(),
+        new Veneration(),
+        new Groundwork(),
+        new CarefulSynthesis()
+      ],
+      generateStats(80, 2450, 2500, 541)
+    );
+    const result = simulation.run(true);
+
+    expect(simulation.success).toBeFalsy();
+    expect(result.failCause).toBe(
+      SimulationFailCause[SimulationFailCause.MISSING_STATS_REQUIREMENT]
```

Comment:
Can't you just check for `SimulationFailCause.MISSING_STATS_REQUIREMENT` here?

Reply:
```
● Craft simulator tests › Should fail a craft when user is below minimum stat requirements

  expect(received).toBe(expected) // Object.is equality

  Expected: 4
  Received: "MISSING_STATS_REQUIREMENT"```